test/basic-test.html

```diff
@@ -431,6 +437,18 @@
     });
   });
 
+  suite('remove key behavior with null target', function () {
+    setup(function () {
+      container = fixture('NoIronA11yKeysBehavior');
+    });
+    test('add and remove a iron-a11y-keys-behavior', function (done) {
+      var element = document.createElement('x-a11y-basic-keys');
+      element.keyEventTarget = null;
+      Polymer.dom(container).appendChild(element);
```

Comment:
You could add this to `document.body`, so you wouldn't need to have all this setup for the container. Remember to remove it from document.body after you're done with the test.